src/rules/07_find_pauses.snake

```diff
@@ -6,34 +6,29 @@
 # 200 bp window
 PAUSE_SEP = 30
 
-rule find_pauses:
+rule find_pauses_pos:
     input:
         pos_bg = RESULTS + "/{sam_grp}/{sam_grp}_pos.bedgraph.gz",
         neg_bg = RESULTS + "/{sam_grp}/{sam_grp}_neg.bedgraph.gz"
```

Comment:
Why is the `find_pauses` rule split into separate positive and negative rules? If we do this, `neg_bg` can be excluded from the input for the `find_pause_pos` rule

Reply:
Ohh I see now, this is to speed up the rule, we should just remove the unneeded inputs from each pos/neg rule